indico/modules/events/surveys/models/anonymous_submissions.py

```diff
@@ -0,0 +1,61 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2021 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from indico.core.db import db
+
+
+class AnonymousSurveySubmission(db.Model):
+    __tablename__ = 'anonymous_submissions'
+    __table_args__ = {'schema': 'event_surveys'}
+
+    #: The ID of the survey
+    survey_id = db.Column(
+        db.ForeignKey('event_surveys.surveys.id', ondelete='CASCADE'),
+        primary_key=True,
+        index=True
+    )
+    #: The ID of the user
+    user_id = db.Column(
+        db.ForeignKey('users.users.id', ondelete='CASCADE'),
+        primary_key=True,
+        index=True,
+    )
+    #: The Survey being submitted
```

Comment:
```suggestion

    #: The Survey being submitted
```

(BTW, feel free to omit docstrings in simple models, pretty much all those attributes are self-explanatory and we stopped adding those verbose docstrings pretty quickly during the 2.0 progress)